spec/1.1/index.html

```diff
@@ -1437,10 +1437,10 @@ <h2 class="no-num no-toc no-ref heading settled" id="subtitle"><span class="cont
      <dd class="editor p-author h-card vcard"><a class="p-name fn u-email email" href="mailto:kgilbert@mozilla.com">Kearwood Gilbert</a> (<a class="p-org org" href="https://mozilla.org/">Mozilla</a>)
      <dd class="editor p-author h-card vcard"><a class="p-name fn u-email email" href="mailto:cvan@mozilla.com">Chris Van Wiemeersch</a> (<a class="p-org org" href="https://mozilla.org/">Mozilla</a>)
      <dt>Participate:
-     <dd><span><a href="https://github.com/w3c/webvr/issues/new">File an issue</a> (<a href="https://github.com/w3c/webvr/issues">open issues</a>)</span>
-     <dd><span><a href="https://lists.w3.org/Archives/Public/public-webvr/">Mailing list archive</a></span>
-     <dd><span><a href="irc://irc.w3.org:6665/">W3C’s #webvr IRC</a></span>
-     <dd><span></span>
+     <dd><a href="https://github.com/w3c/webvr/issues/new">File an issue</a> (<a href="https://github.com/w3c/webvr/issues">open issues</a>)
```

Comment:
These sort of "unnecessary" changes are the result of Bikeshed updates and can't really be avoided, so don't worry about them getting rolled into you patch.